src/app/content/reducer.ts

```diff
@@ -88,7 +88,7 @@ function reduceContent(state: State, action: AnyAction) {
       return reduceReceiveBook(state, action);
     }
     case getType(actions.requestPage):
-      return {...state, loading: {...state.loading, page: action.payload}};
+      return {...omit(['page'], state), loading: {...state.loading, page: action.payload}};
```

Comment:
similar to the way this is done down on line 118, i think we only want to omit the page if the requested page is different from the current one